### DISCUSSION TITLE
[Feature] By-Library transcoding settings

### DISCUSSION THREAD

**Discussion by CommanderRedYT** [immich-app/immich#8845] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be nice to be able to toggle transcoding and maybe change the settings per library.
I would like to turn off transcoding, but just for one external library of action cam videos. I just want to use immich for viewing the images in there and for playback of too large videos I can do that through local network.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by skatsubo** [comment]
Seems this feature will not be implemented. See https://github.com/immich-app/immich/pull/20797
> Thank you for the PR, but I am pretty positive that we don't want to handle this granularity behavior for the transcoding process.

**Comment by CommanderRedYT** [comment]
Do you have some kind of workaround to this? I really need this to be able to use immich for everything, having a 2nd instance or another program doesn't sound nice...

**Reply by thegabriele97** [reply]
> Do you have some kind of workaround to this? I really need this to be able to use immich for everything, having a 2nd instance or another program doesn't sound nice...

I think it's confirmed that having a 2nd instance or another program will be the only way to achieve this :(
